apps/modclock/app.js

```diff
@@ -9,7 +9,21 @@ Graphics.prototype.setFontBold = function(scale) {
 { // must be inside our own scope here so that when we are unloaded everything disappears
   // we also define functions using 'let fn = function() {..}' for the same reason. function decls are global
 let drawTimeout;
+  
+let calcStrLength=function(str,maxLength){
```

Comment:
Looks good in general. At the moment, for this function, on reading `calcStrLength`, a reader might expect it to return a number - perhaps a name like `limitStrLength` might get across the intention, wdyt?